### PR TITLE
Feature/detail type kwarg

### PR DESCRIPTION
**Pull request by defond0**
Just wanted to add a detail_type kwarg to to make that a configurable field

### CODE REVIEW

File: marquee/handler.py [hunk 1]
```diff
@@ -7,6 +7,20 @@
 
 
 class CloudWatchEventsHandler(Handler):
+
```
Comment (theherk) [inline marquee/handler.py:10]: Remove this extra line pretty please.

File: marquee/handler.py [hunk 2]
```diff
@@ -7,6 +7,20 @@
...
+        Will set detail_type on handler object then invoke Handler __init__
+        with any passed in arguments
+        """
+        self.detail_type=detail_type
+        super(CloudWatchEventsHandler, self).__init__(
+            *args,
+            **kwargs
+        )
+
+
```
Comment (theherk) [inline marquee/handler.py:19]: Same here, extra line.

File: marquee/handler.py [hunk 3]
```diff
@@ -7,6 +7,20 @@
...
+        """Overridden __init__ method.
+
+        Will set detail_type on handler object then invoke Handler __init__
+        with any passed in arguments
+        """
+        self.detail_type=detail_type
+        super(CloudWatchEventsHandler, self).__init__(
+            *args,
+            **kwargs
+        )
```
Comment (theherk) [inline marquee/handler.py]: I think this super call would look prettier on one line.

File: marquee/handler.py [hunk 4]
```diff
@@ -15,12 +29,13 @@ def emit(self, record):
...
         client = boto3.client('events')
         source = self.formatter.source if isinstance(self.formatter, MarqueeFormatter) else __name__
+        detail_type = self.detail_type if self.detail_type is not None else 'marquee'
         return client.put_events(
             Entries=[
                 {
                     'Time': self.formatter.formatTime(record),
                     'Source': source,
-                    'DetailType': 'marquee',
+                    'DetailType': detail_type,
```
Comment (theherk) [inline marquee/handler.py:34]: Making this configurable is great.

**Review by theherk — CHANGES_REQUESTED** [review]
Looks great. You may want to add this option to the usage.rst documentation. Thanks for the contribution.

File: marquee/handler.py (hunk 4 above)
Comment (defond0) [reply marquee/handler.py:34]: so it is said, so shall it be done

**Review by theherk — APPROVED** [review]

**Pull request merged**